modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -469,6 +469,27 @@ The `Iterator` type is based on the https://docs.oracle.com/javase/8/docs/api/ja
 
 Note that like the Java class, the iterator is designed to be consumed only once. For example, if you then pass this value to a logger would result in consuming it and it would no longer be readable to further elements in the flow.
 
+[[dw_type_null]]
+== Null (dw::Core Type)
+
+DataWeave defines a `Null` data type for `null` values. It is important to note that `null` 
+is the value, while `Null` is the type. DataWeave functions do not recognize `Null` as
+a `null` value. 
+
+You can determine whether a DataWeave function can handle `null` values by
+looking in the DataWeave reference documentation for a function signature
+that handles a `Null` type. For example, the
+xref:dw-core-functions-flatten.adoc#flatten2[flatten] function has two function
+signatures, one of which is `flatten(Null): Null`. The signature indicates that
+`flatten` can handle `null` values. On the other
+hand, xref:dw-core-functions-plusplus.adoc[++] (concatenate), which lacks
+a signature for handling `Null`, cannot handle `null` values.
+
+* `flatten([[1],2,[null],null])` returns `[1,2,null,null]`
+* `"a" &#43;&#43; null` returns the error
+`Unable to call \`&#43;&#43;\` with (`String`, `Null`)`.
```

Comment:
Are the backticks in the correct place here: \`&#43;&#43;\`

Reply:
fixed